metron-interface/metron-alerts/src/app/utils/elasticsearch-utils.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {ColumnMetadata} from '../model/column-metadata';
+import {AlertsSearchResponse} from '../model/alerts-search-response';
+
+export class ElasticsearchUtils {
+
+  public static excludeIndexName = 'kibana';
+
+  private static createColumMetaData(properties: any, columnMetadata: ColumnMetadata[], seen: string[]) {
+     try {
+       let columnNames = Object.keys(properties);
+       for (let columnName of columnNames) {
+         if (seen.indexOf(columnName) === -1) {
+           seen.push(columnName);
+           columnMetadata.push(
+             new ColumnMetadata(columnName, (properties[columnName].type ? properties[columnName].type.toUpperCase() : ''))
```

Comment:
Forcing upper case in this model prevents sorting from working later... e.g. sorting by threat.triage.level leads to failed searches with: Caused by: java.lang.IllegalArgumentException: No mapper found for type [DOUBLE] 

The type mapping in elastic appears to be case sensitive, so squashing case here is going to break things.